src/nlc/NLC.json

```diff
@@ -13,6 +13,16 @@
 			"help"
 		  ]
 	    },
+			{
+	      "class": "nlc.management.help",
+	      "description": "Get help for managing classifiers",
+	      "emittarget": "nlc.management.help.js",
+	      "texts": [
+			"Can you help me with nlc?",
+			"Help me with my classifiers",
+			"I need help with nlc"
```

Comment:
Remove generic phrases `Can you` and `I need`.
Replace one of the instances of "nlc" with "natural language". So the last statement becomes "Help with natural language`